Fixes #235 changed code of conduct text and link to participation guildlines

jayyyin opened the pull request:
do note I have changed the Class name here so this issue https://github.com/mozilla/mofo-bootstrap/issues/122 has to be fixed first before we pull this

jayyyin commented:
The issue this fixes is here https://github.com/mozilla/foundation.mozilla.org/issues/235

gideonthomas commented on source/pug/templates/master.pug:
```diff
@@ -78,8 +78,8 @@ html
               a(href="https://creativecommons.org/licenses/by/4.0" class="footer-link-cc-license") License
-            //- li.col-auto
-            //-   a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            - li.col-auto
```
remove the `-` on both lines

gideonthomas commented on source/pug/templates/master.pug:
```diff
@@ -78,8 +78,8 @@ html
-            //- li.col-auto
-            //-   a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            - li.col-auto
+            -   a(href="https://www.mozilla.org/en-US/about/governance/policies/participation/" class="footer-link-participation-guidelines") Participation Guidelines
```
remove the `/en-US` part of the URL.

gideonthomas replied:
also, let's leave the class name as what it was before.

When we land your other PR, we will need to update the version of mofo-bootstrap here so we can change the class name here in that patch.

jayyyin replied:
ok

jayyyin commented:
@gideonthomas alright so I've removed the en-gb part of the url and reverted the class name for now, when we get around to updating the required package I'll make another issue/PR

gideonthomas commented:
Thanks @jayyyin, this looks right to me. I'll let @gvn or @mmmavis merge it in if they're ok with it as well.

gvn commented:
LGTM. Thanks!

jayyyin commented:
@gvn what's LGTM mean?
@gideonthomas thanks for the answers for getting started on these issues.

gvn commented:
"Looks good to me". People often write that when they approve code for merging, similar to "R+".

jayyyin commented:
@gvn I see, thanks man
